### ISSUE TITLE
Demo not working? geo.applyMatrix4 is not a function

### ISSUE THREAD

**Issue by ccj242** [jsantell/THREE.IK#27]
Hi! This looks like an amazing library and it's definitely possible I'm missing something here, but I can't get the demo working -- I get "geo.applyMatrix4 is not a function" error in the console with all browsers. Thanks!

**Comment by ccj242** [comment]
Anyone have ideas on this? I can't even get the online demo working: https://jsantell.github.io/THREE.IK/

Thanks in advance for any advice!

**Comment by jamesrez** [comment]
@ccj242  Yeah I'm also seeing some issues with the demos. I'm guessing this library needs to be updated for the latest version of THREE.js

I hope it's a quick fix that @jsantell can take care of as I'd like to use this for my project,

**Comment by ccj242** [comment]
@jamesrez - have you had any luck downgrading three.js to previous versions? I'm trying that now and no luck so far, but I'll let you know if I get anywhere. Thanks for confirming! I thought it was just me and I was going a bit mad!

**Comment by jamesrez** [comment]
@ccj242 I've been trying to get it working in the latest version and it's certainly broken right now.

After debugging some of the code I was able to get it kind of working with my model, but all of the bones were super stretched out and twisty no matter how much I played around with the values.

I need IK though, so I'm still looking for a fix.

**Comment by ccj242** [comment]
So per [this PR](https://github.com/jsantell/THREE.IK/pull/26) I was able to get the examples working by installing version 127 of three.js using **install three@0.127.0 --save** Hopefully this helps!

**Comment by drone1** [comment]
The demo linked from the README ([https://jsantell.github.io/THREE.IK/](https://jsantell.github.io/THREE.IK/)) appears to be out of date and still have this issue.

**Comment by icejix** [comment]
I'm looking for a three.js IK library, and this one seemed good, but I did assume the last modified date being years ago would be an issue. I really hope there are plans to fix it.

**Comment by pedrorolosilva** [comment]
2025 and I also need this. @jsantell is anyone working on this? Keep up the great work!
